index.src.html

```diff
@@ -501,7 +501,8 @@ <h3 id="dom-api">JavaScript API</h3>
       Clears data based on the values in the |options| argument.
       Returns a Promise that resolves when clearing is complete. If no
       {{StorageClearOptions/types}} are specified, all data types will be
-      cleared.
+      cleared. See [[#clear-api]] for the formal description of the clearing
+      algorithm.
```

Comment:
FWIW, I think the way you're using Bikeshed by referencing algorithms through section identifiers rather than their actual definitions is somewhat of an abuse.